examples/basic.html

```diff
@@ -8,11 +8,18 @@
     <script class='remove'>
       var respecConfig = {
         specStatus: "ED",
+        subtitle: "Subtitle",
+        shortName: "shortname-api",
         editors: [{
           name: "Your Name",
           url: "https://your-site.com",
         }],
+        authors: [{
```

Comment:
We probably want to exclude authors... in the standards community, editors are actually authors. Classic read email the role of an editor within the W3C community: 
https://lists.w3.org/Archives/Member/chairs/1999JanMar/0056